jax/_src/numpy/lax_numpy.py

```diff
@@ -3106,7 +3359,7 @@ def _concatenate_array(arr, axis: int):
 
 @_wraps(np.concatenate)
 def concatenate(arrays, axis: int = 0):
-  if isinstance(arrays, ndarray):
+  if isinstance(arrays, (np.ndarray, ndarray)):
```

Comment:
Here and elsewhere where `isinstance` is used, it might be more correct to use `if _arraylike(arrays)`, though perhaps that should be a separate PR because it does slightly change the behavior.

Reply:
Yes, I think I'd rather keep this change mechanical and think about that in a future change.